test/integration/runner.js

```diff
@@ -48,7 +48,6 @@ for (const testName of readdirSync(__dirname)) {
     const actualWebDependenciesLoc = path.join(__dirname, testName, 'web_modules');
     const expectedWebDependencies = await fs.readdir(expectedWebDependenciesLoc).catch(() => {});
     if (!expectedWebDependencies) {
-      assert.rejects(() => fs.readdir(actualWebDependenciesLoc), 'web_modules/ exists');
```

Comment:
@FredKSchott just want to confirm this is an OK change for the test runner? That means now if `expected-install/` is missing, we’ll assume it was intentional.

Reply:
lets make this explicit. How about an array of test names that this is okay for in the test runner.

```
const SKIP_FILE_CHECK = ['config-rollup'];
```